ios/Podfile

```diff
@@ -21,7 +21,7 @@ target 'BitriseReactNativeSample' do
     #
     # Note that if you have use_frameworks! enabled, Flipper will not work and
     # you should disable the next line.
-    :flipper_configuration => FlipperConfiguration.enabled,
+    :flipper_configuration => FlipperConfiguration.disabled,
```

Comment:
Do we need the magic sed script if this is disabled here?

Reply:
So Flipper is a tool used for debugging react native apps.

Since we're not debugging we don't need it on the CI and it actually makes the build much slower to install all these pods and then build them.

The Podfile can be regenerated by the react-native command that's why I included the magic sed command in the yml.

Maybe I should revert the change of disabling Flipper in the Podfile as it's more representative of a default user project setup 🤔

Reply:
Sounds like I should actually remove the magic script and modify Flipper's use in the Podfile instead per Grayson's input in our weekly demo.

Reply:
44bfcd0